src/server/admin/models.py

```diff
@@ -134,23 +122,23 @@ class UserRole(Base):
     name = Column(String(50), unique=True)
 
     @staticmethod
-    def create_role(name: str) -> None:
+    @default_session(WriteSession)
+    def create_role(name: str, session) -> None:
         get_structured_logger("api_user_models").info("creating user role", role=name)
-        with Session() as session:
-            session.execute(
-                f"""
+        # TODO: check role doesnt already exist
```

Comment:
¯\\\_(ツ)_/¯ im ok with neither, esp because `create_role()` isnt even used anywhere at the moment.  if the role already exists, this is a no-op -- depending on the intention of the caller, that could be good or bad.  so, idk...  i say leave it as-is.